acestep/constants.py

```
@@ -84,7 +84,9 @@
 # - extract: Separate individual tracks/stems from audio
 # - lego: Multi-track generation (add layers)
 # - complete: Automatic completion of partial audio
-TASK_TYPES_BASE = ["text2music", "repaint", "cover", "cover-nofsq", "extract", "lego", "complete"]
+# - edit: Flow-edit — morph the source toward a new prompt/lyrics
+#   (paired CFG, ~4× decoder forwards/step; only on base variants)
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)